package-lock.json

```diff
@@ -2180,7 +2180,7 @@
     },
     "@types/node": {
       "version": "8.9.5",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-8.9.5.tgz",
+      "resolved": "http://registry.npmjs.org/@types/node/-/node-8.9.5.tgz",
```

Comment:
Please revert package-lock changes.

Reply:
When you revert the package-lock changes you may need to merge the `golden/clr-angular.d.ts` and/or rerun `npm run golden:fix`. Sorry about that we had a couple other PRs merge in that updated the same file. Other that that it looks great!

Reply:
`package-lock.json` changes are now fixed, I rerun `npm run golden:fix` and it looks like there's no changes.

Reply:
Edit: also merged in the changes with golden from master.